docs/page.md

```diff
@@ -157,6 +158,7 @@ A section is a reusable piece or part of a page.
 
 Attributes|Type|Use|Description
 ---|---|---|---
+`deprecated`|string|optional|Used to warn about the future deprecation of the section. String will appear in Allure reports and console output at runtime.
```

Comment:
Section reference in page do not have `deprecated` attr
It is only available under Section entity.

Reply:
Please remove this from Page doc. This attr not exist here.